src/models/slices/directanswer.ts

```diff
@@ -1,5 +1,12 @@
 import { FeaturedSnippetDirectAnswer, FieldValueDirectAnswer } from '@yext/answers-core';
 
+/**
+ * Maintains the direct answer associated with the latest search.
+ */
 export interface DirectAnswerState {
+  /**
+   * The data for the direct answer. The type of the data is based on where the
+   * answer was found by the Answers API.
```

Comment:
I think we should provide a bit more detail on the possible sources of data here.

Reply:
Something like this? "The type of the data is determined by the Answers API based on whether the answer was found within a document or was a field value in the knowledge graph."

Reply:
Sounds good!

Reply:
Updated!